superset-frontend/cypress-base/cypress/integration/dashboard/load.js

```diff
@@ -31,16 +31,12 @@ export default () =>
 
       cy.get('#app').then(data => {
         const bootstrapData = JSON.parse(data[0].dataset.bootstrap);
-        const dashboardId = bootstrapData.dashboard_data.id;
         const slices = bootstrapData.dashboard_data.slices;
         // then define routes and create alias for each requests
         slices.forEach(slice => {
           const alias = `getJson_${slice.slice_id}`;
           const formData = `{"slice_id":${slice.slice_id}}`;
-          cy.route(
```

Comment:
why are you removing dashboardId here?

Reply:
This test case only interacts with one dashboard. A more relaxed URL pattern makes it more resilient to future unsubstantial change. It also keeps the code simpler.